group_vars/gxconfig.yml

```diff
@@ -2589,12 +2642,15 @@ base_app_main: &BASE_APP_MAIN
   # as threshold (above threshold: regular select fields will be used)
   #select_type_workflow_threshold: -1
 
-  # API key for OpenAI (https://openai.com/) to enable the wizard (or
-  # more?)
+  # API key for an AI provider. AI provider is Openai By default
+  # (https://openai.com/) to enable the wizard (or more?)
   openai_api_key: "{{ openai_api_key }}"
 
-  # OpenAI model to enable the wizard.
-  #openai_model: gpt-4o
+  # AI API base URL. Needs to be OpenAI compatible, defaults to OpenAI
+  #ai_api_base_url: null
```

Comment:
@anuprulez we have this setting now. So if we can use the University server let me know.

Reply:
KI support did not reply about the meeting proposal. If I have an update, I will share